util/findLostPatchesOneRepository.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/sh
+
+###########################################
+#  
+#   Find any missing commits that are IN a given branch 
+#   but are NOT in the 2nd branch. 
+#  
+#   These missing commits are found via a concept
```

Comment:
some of us just fancy by nature. Must be my New York tax-hiking, government-expanding, latte-drinking, sushi-eating, Volvo-driving, New York Times-reading, body-piercing, Hollywood-loving, left-wing freak show values.